### PR TITLE
Updates to prepare for StaPH-B ToolKit V2

### PR DESCRIPTION
**Pull request by k-florek**
1. Added configuration to allow nextflow to run via github repository
2. Fixed minor bug in config template
3. Added pangolin and nextclade version columns to result summary

### CODE REVIEW

**Review by erinyoung — COMMENTED** [review]
Does anything need to be in `configs/cecret_config_template.config` for this to work?

File: configs/cecret_config_template.config [hunk 1]
```diff
@@ -303,7 +303,7 @@ process {
 
   withName:iqtree2{
    cpus = 8
-   container = iqtree_container
+   container = iqtree2_container
```
Comment (erinyoung) [inline configs/cecret_config_template.config:306]: This is embarrassing. Thank you for catching this.

**Comment by erinyoung** [comment]
I'm running this through our test dataset as we speak. I don't have fancy AWS support to try to do what you did with github actions.

**Comment by k-florek** [comment]
```
Does anything need to be in configs/cecret_config_template.config for this to work?

I ask because I am going to use Cecret as a template to get another workflow into the staphB toolkit.
```
It might be easier to have a quick call but yes you need to have a configuration file set for the workflow to run properly.

```
I'm running this through our test dataset as we speak. I don't have fancy AWS support to try to do what you did with github actions.
```
You can setup github actions to run the jobs on an on-prem server/workstation. It would just require installing some software on the server that would poll github every few minuets and look for new actions that need to be run. Not sure if that is feasible for you but just a thought.

**Comment by erinyoung** [comment]
> You can setup github actions to run the jobs on an on-prem server/workstation.

So I've heard. I really want to try getting something like that going. It'd really simplify pull requests and would reduce errors. I just need some time to play around with it.

**Comment by erinyoung** [comment]
Everything checks out. Thank you!

**Pull request merged**